package-lock.json

```diff
@@ -1,22 +1,26 @@
 {
   "name": "boring-vault-ui",
-  "version": "1.8.32",
+  "version": "1.9.0",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "boring-vault-ui",
-      "version": "1.8.32",
+      "version": "1.9.0",
       "license": "MIT",
       "dependencies": {
         "@chakra-ui/react": "^2.8.2",
+        "@coral-xyz/anchor": "^0.31.1",
         "@emotion/react": "^11.11.4",
         "@emotion/styled": "^11.11.5",
         "@ethersproject/bytes": "^5.8.0",
         "@safe-global/safe-apps-provider": "^0.18.3",
         "@safe-global/safe-apps-sdk": "^9.1.0",
+        "@solana/spl-token": ">=0.1.8 <0.3.0",
+        "@solana/web3.js": ">=1.73.0 <2.0.0",
         "@wagmi/connectors": "^5.7.13",
         "@wagmi/core": "^2.17.0",
+        "bn.js": ">=5.1.0 <6.0.0",
```

Comment:
Could also use bignumber from ethers if you wanted 1 less dependency, not a hard req just something to consider